### PR TITLE
add status_page_one function

### PR DESCRIPTION
**Pull request by lifeblood**
Hello, my name is Shao Dong, I recently in the work with your products, in order to work need me add a function(status_page_one), you see, is not useful, can be merge

### CODE REVIEW

**Comment by agentzh** [comment]
@lifeblood Could you describe the purpose and usage of your new addition? Why should we have it? And how are we going to use it?

**Comment by lifeblood** [comment]
generate an HTML page for specified upstream
for example: http://domain.com/status_one?upstream=foo
generate foo upstream status page


            location /status_one {
                access_log off;
                default_type text/plain;
                content_by_lua_block {
                        local hc = require "resty.upstream.healthcheck"
                        ngx.say("Nginx Worker PID: ", ngx.worker.pid())
                        ngx.print(hc.status_page_one())
                }
            }

File: lib/resty/upstream/healthcheck.lua [hunk 1]
```diff
@@ -701,4 +701,57 @@ function _M.status_page()
     return concat(bits)
 end
 
+function _M.status_page_one()
```
Comment (agentzh) [inline lib/resty/upstream/healthcheck.lua]: I'm not happy with the function name. Maybe we should reuse the existing `status_page` function and just make it accept a function parameter? It's more flexible than hard-coding the "upstream" URI arg name in the implementation I think.

Comment (lifeblood) [inline lib/resty/upstream/healthcheck.lua]: You mean is status_page(parameter)?

Comment (agentzh) [inline lib/resty/upstream/healthcheck.lua]: Yes, I mean `status_page(upstream_name)` where the parameter `upstream_name` is optional.

**Comment by lifeblood** [comment]
get_upstreams() can be obtain one upstream?

**Comment by agentzh** [comment]
@lifeblood You can use `if` statement to branch the code, no?

**Comment by lifeblood** [comment]
Ok

**Comment by lifeblood** [comment]
have done, pls check.

**Comment by lifeblood** [comment]
Is there any place to improve?

**Pull request closed without merging**